app/controllers/runtime/organizations_controller.rb

```diff
@@ -285,9 +285,26 @@ def user_guid_parameter
     end
 
     def after_create(organization)
-      return if SecurityContext.admin?
-      organization.add_user(user)
-      organization.add_manager(user)
+      unless SecurityContext.admin?
+        organization.add_user(user)
+        organization.add_manager(user)
+      end
+
+      organization.users.each do |user|
+        @user_event_repository.record_organization_role_add(organization, user, 'user', UserAuditInfo.from_context(SecurityContext), request_attrs)
+      end
+
+      organization.auditors.each do |auditor|
```

Comment:
Spoke with @ScarletTanager on Slack, I definitely misunderstood the change. It seems that during org creation, we can also supply params `billing_manager_guids` (not documented). I initially believed we only had to emit events for the two add_user/add_manager calls.

Merging once I fix the api_version spec.